locale/en_GB/LC_MESSAGES/django.po

```diff
@@ -197,9 +197,11 @@ msgstr "Search"
 msgid "basicsearchform.placeholder"
 msgstr "Search..."
 
+
+
 #: ds_judgements_public_ui/templates/includes/search_term_component.html
 msgid "basicsearchform.helptext"
-msgstr "Enter keyword, party name or neutral&nbsp;citation"
+msgstr "Enter keyword&#44; party name neutral&nbsp;citation or phrases surrounded by quote marks e.g. &#34;blade crime&#34;"
```

Comment:
the comma should be absolutely fine as-is.
I'd prefer `&quot;` to `&#34;` if `'..."blade crime"'` doesn't work.
Blade crime is a bit of a meme amongst ourselves, but maybe something more generic would be useful: `"statutory instrument"`, perhaps?